src/main/java/org/bptlab/cepta/StreamingJob.java

```diff
@@ -38,27 +39,12 @@ public static void main(String[] args) throws Exception {
     // set up the streaming execution environment
     final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 
-    /*
-     * Here, you can start creating your execution plan for Flink.
-     *
-     * Start with getting some data from the environment, like
-     * 	env.readTextFile(textPath);
-     *
-     * then, transform the resulting DataStream<String> using operations
-     * like
-     * 	.filter()
-     * 	.flatMap()
-     * 	.join()
-     * 	.coGroup()
-     *
-     * and many more.
-     * Have a look at the programming guide for the Java API:
-     *
-     * http://flink.apache.org/docs/latest/apis/streaming/index.html
-     *
-     */
+    // create mock data stream
+    DataStream<Actor> actors = env.fromElements(new Actor(1, null));//, new Actor(2, "Aaaron"));
```

Comment:
Mock data belonging to tests should not be within the normal code base, should it?